functions/validating_inputs.rb

```diff
@@ -0,0 +1,35 @@
+def info_request(question)
+  puts question
+  gets.chomp
+end
+
+def given_name(input)
```

Comment:
przyjmujemy konwencję zgodnie z którą nazwy metod zwracających boolean, kończymy znakiem zapytania
https://github.com/rubocop-hq/ruby-style-guide#bool-methods-qmark

Reply:
i w ogóle fajnie by było metody walidacji przenieść do osobnego obiektu, na przykład:
```ruby
class Validator
  def initialize(first_name:, last_name:, zip:, id:)
    @first_name = first_name
    @last_name = last_name
    @zip = zip
    @id = id
  end
  
  def valid?
     first_name_valid? && last_name_valid? (...)
  end

  def errors
    return if valid?
    errors = []
    errors <<  'The first name must be filled in.' unless first_name_valid?
    (...)
    errors
  end

  private

  attr_reader :first_name, :last_name, :zip, :id

  def first_name_valid?
    first_name.size >= 2
  end
end
```

i potem użyć go w main_program. Plus będzie się to super testowało :)